rmf_task_ros2/src/rmf_task_ros2/bidding/Auctioneer.cpp

```diff
@@ -34,22 +54,33 @@ Auctioneer::Implementation::Implementation(
   {
     evaluator = std::make_shared<QuickestFinishEvaluator>();
     RCLCPP_INFO(
-      node->get_logger(),
+      node_logging_interface->get_logger(),
       "Dispatcher evaluator set to QuickestFinishEvaluator by default");
   }
   const auto dispatch_qos = rclcpp::ServicesQoS().reliable();
 
-  bid_notice_pub = node->create_publisher<BidNoticeMsg>(
-    rmf_task_ros2::BidNoticeTopicName, dispatch_qos);
-
-  bid_proposal_sub = node->create_subscription<BidResponseMsg>(
-    rmf_task_ros2::BidResponseTopicName, dispatch_qos,
+  bid_notice_pub = rclcpp::create_publisher<BidNoticeMsg>(
+    node_parameters_interface,
+    node_topics_interface,
+    rmf_task_ros2::BidNoticeTopicName,
+    dispatch_qos);
+
+  bid_proposal_sub = rclcpp::create_subscription<BidResponseMsg>(
+    node_parameters_interface,
+    node_topics_interface,
+    rmf_task_ros2::BidResponseTopicName,
+    dispatch_qos,
     [&](const BidResponseMsg::UniquePtr msg)
     {
       this->receive_response(*msg);
     });
 
-  timer = node->create_wall_timer(std::chrono::milliseconds(200), [&]()
+  timer = rclcpp::create_timer(
```

Comment:
It seems this changed the function from `create_wall_timer` to `create_timer`, there is a [rclcpp::create_wall_timer](https://github.com/ros2/rclcpp/blob/db8d917a12d640b832cf2ee773e338b98b69e191/rclcpp/include/rclcpp/create_timer.hpp#L191), any reason for this change?

Reply:
Thanks for catching that, I intended to "fix" the behavior, since a generic timer adheres to sim time as well, so that might be what we actually want. But forgot to revert this to not change any behavior. I'll switch it back to wall timer

Reply:
https://github.com/open-rmf/rmf_ros2/pull/420/commits/e80451e22e8457f8707d9e82c218deb0fbb482bf